libs/dataviews-client/src/resolve-dataviews-generators.ts

```diff
@@ -545,28 +554,29 @@ export function getDataviewsGeneratorConfigs(
     {
       activityDataviews: [] as UrlDataviewInstance[],
       detectionDataviews: [] as UrlDataviewInstance[],
+      trackDataviews: [] as UrlDataviewInstance[],
       otherDataviews: [] as UrlDataviewInstance[],
     }
   )
 
   // If activity heatmap animated generators found, merge them into one generator with multiple sublayers
   const mergedActivityDataview = activityDataviews?.length
     ? getMergedHeatmapAnimatedDataview(activityDataviews, {
-        ...params,
-        mergedHeatmapGeneratorId: MERGED_ACTIVITY_ANIMATED_HEATMAP_GENERATOR_ID,
-      })
+      ...params,
+      mergedHeatmapGeneratorId: MERGED_ACTIVITY_ANIMATED_HEATMAP_GENERATOR_ID,
+    })
     : []
   const mergedDetectionsDataview = detectionDataviews.length
     ? getMergedHeatmapAnimatedDataview(detectionDataviews, {
-        ...params,
-        mergedHeatmapGeneratorId: MERGED_DETECTIONS_ANIMATED_HEATMAP_GENERATOR_ID,
-      })
+      ...params,
+      mergedHeatmapGeneratorId: MERGED_DETECTIONS_ANIMATED_HEATMAP_GENERATOR_ID,
+    })
     : []
-
   const generatorsConfig = [
     ...mergedActivityDataview,
     ...mergedDetectionsDataview,
-    ...otherDataviews,
+    ...trackDataviews,
+    ...otherDataviews
```

Comment:
Why do we need to separate `trackDataviews` if we don't do any change on them and finally we concat them? They were previously being assigned inside `otherDataviews`